Makefile

```diff
@@ -5,7 +5,7 @@ GOGET=${GOCMD} get
 GOTEST=${GOCMD} test
 GORUN=${GOCMD} run 
 
-BINARY_NAME=bin/guest-agent-test-extension
+BINARY_NAME=bin/GuestAgentTestExtension
```

Comment:
why did you change the name? Any particular reason?

Reply:
None of the other extensions had this format for their name so I wanted to match that

Reply:
Ahh ok makes sense